integration/signer/tests/test-bypass-and-sign-with-kms.sh

```diff
@@ -41,5 +43,11 @@ trap 'delete_occ $GOOD_IMG_DIGEST_URL'  EXIT
 -policy=policy.yaml \
 -note_name=${NOTE_NAME}
 
+# deploy to a binauthz-enabled cluster signer-int-test
+clean_up() { ARG=$?; delete_image $GOOD_IMAGE_URL; delete_occ $GOOD_IMG_DIGEST_URL; delete_pod signer-int-test-pod; exit $ARG;}
```

Comment:
I'm not thinking too much, but it feels that using || could be complicated because you also need to consider clean up for success cases?

Ack on documentation.